pkgs/by-name/al/alist/update.nix

```diff
@@ -0,0 +1,50 @@
+{
+  writeShellApplication,
+  lib,
+  nix,
+  nix-update,
+  curl,
+  coreutils,
+  gnused,
+  jq,
+  common-updater-scripts,
+}:
+
+lib.getExe (writeShellApplication {
+  name = "update-alist";
+  runtimeInputs = [
+    coreutils
+    curl
+    gnused
+    jq
+    nix
+    common-updater-scripts
+    nix-update
+  ];
+
+  text = ''
+    set -euo pipefail
```

Comment:
writeShellApplication will set shell options according to its `bashOptions` config.  It seems `set -euo pipefail` is the same as the default value of `bashOptions` so this line can be removed.